messages/retrieve.metadata.md

```diff
@@ -94,3 +94,35 @@ Number of minutes to wait for the command to complete and display results to the
 # flags.wait.description
 
 If the command continues to run after the wait period, the CLI returns control of the terminal window to you.
+
+# RetrieveTitle
+
+Retrieving metadata
+
+# Pending
+
+Pending
+
+# InProgress
+
+In Progress
+
+# Succeeded
+
+Succeeded
+
+# Cancelled
+
+Cancelled
```

Comment:
```suggestion
Canceled
```

Reply:
NOTE: Both 'canceled' and 'cancelled" are correct, but the former is more used in the US, so let's stick with it.